content/02.body.md

```diff
@@ -64,6 +64,27 @@ Further discussion of this approximate edge prior and an derivation are availabl
 
 ### Prediction tasks
 
+We performed three prediction tasks to assess the performance of the edge prior, and we compared two additional features, a scaled product of source and target degree (scaled to the range [0, 1]), and our approximation of the edge prior.
+For the first two prediction tasks, we used 20 biomedical networks from the Hetionet heterogeneous network [@doi:10.7554/eLife.26726] that had at minimum 2000 edges.
+In the first task, we computed the degree-based prediction features (edge prior, scaled degree product, and prior approximation), and predicted the original edges in the network.
+Negative samples in the first task were node pairs in which an edge was not present in the original network.
```

Comment:
```suggestion
We used node pairs that lacked edge in the original network as negative examples.
```
What are positives? I'm imagining those that are present, but perhaps add a sentence to specifically state it.